### PR TITLE
Write patterns deployment

### PR DESCRIPTION
**Pull request by balegas**
Deployment and dockers files for hosting Write patterns example.

I did some changes to load env vars and add db ids, etc.

There seems to be some problem with pattern 4, but I'm not sure if is just me.

### CODE REVIEW

**Comment by balegas** [comment]
This should generally be working, but I wasn't able to test it. Opening PR for visibility and for someone to take over if necessary.

File: examples/write-patterns/sst.config.ts [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    const databaseUri = getNeonDbUri(project, db, false)
+    try {
+      databaseUri.apply(applyMigrations)
+
+      const electricInfo = databaseUri.apply((uri) =>
+        addDatabaseToElectric(uri)
+      )
+
+      const vpc = new sst.aws.Vpc(`write-patterns-${$app.stage}-vpc`)
+      const cluster = new sst.aws.Cluster(
```
Comment (KyleAMathews) [inline examples/write-patterns/sst.config.ts:47]: let's reuse the our dev cluster from cloud

**Review by KyleAMathews — CHANGES_REQUESTED** [review]

File: examples/write-patterns/sst.config.ts (hunk 1 above)
Comment (balegas) [reply examples/write-patterns/sst.config.ts:47]: Alright. Well have to do that though all examples

Comment (KyleAMathews) [reply examples/write-patterns/sst.config.ts:47]: that'd be good — they're slow to setup & fairly expensive

File: examples/write-patterns/patterns/1-online-writes/index.tsx [hunk 2]
```diff
@@ -19,6 +20,8 @@ export default function OnlineWrites() {
     url: `${ELECTRIC_URL}/v1/shape`,
     params: {
       table: 'todos',
+      database_id: import.meta.env.VITE_ELECTRIC_DATABASE_ID,
```
Comment (thruflo) [inline examples/write-patterns/patterns/1-online-writes/index.tsx:23]: The challenge here is that this complicates the example code just for our deployment purposes.

One request is please can you follow the pattern of declaring a `const `DATABASE_URL ...` next to `ELECTRIC_URL` above and providing a default value there. It keeps the actual shape definition cleaner.

Comment (balegas) [reply examples/write-patterns/patterns/1-online-writes/index.tsx]: I moved the params to the top but is not great. I suggest you change it to your taste.

In general is a bit of a pain that we need to set Cloud params explicitly. One possible way to hide is to pass Cloud in the base url, but I'm not sure if the code picks it up properly

File: examples/write-patterns/patterns/1-online-writes/index.tsx [hunk 3]
```diff
@@ -3,7 +3,17 @@ import { v4 as uuidv4 } from 'uuid'
 import { useShape } from '@electric-sql/react'
 import api from '../../shared/app/client'
 
-const ELECTRIC_URL = import.meta.env.ELECTRIC_URL || 'http://localhost:3000'
+const ELECTRIC_URL =
```
Comment (thruflo) [inline examples/write-patterns/patterns/1-online-writes/index.tsx:6]: Maybe we could wrap this up into:

```ts
const { electricUrl, envParams } from '../../shared/app/config'
```

Comment (thruflo) [reply examples/write-patterns/patterns/1-online-writes/index.tsx:6]: I.e.: across all the patterns, given that it's a bunch of noise at the top of the code samples that are all displayed on the page in the Writes guide, etc.

**Review by thruflo — CHANGES_REQUESTED** [review]

**Review by KyleAMathews — APPROVED** [review]
:shipit:

**Pull request merged**